### ISSUE TITLE
Warhammer 40,000: Darktide (Game Pass version) does not track game exit properly

### ISSUE THREAD

**Issue by K900** [JosefNemec/PlayniteExtensions#394]
**Describe the bug**
When starting, then exiting Darktide, Playnite continue to report the game as "running", even though it's not. This seems to be specific to how Darktide does things, as other Windows Store games are tracked correctly.

**Diagnostics ID**
2a16e4ab-4bb5-4ca7-ac1a-d7176db1e9d1

**Comment by JosefNemec** [comment]
Don't own the game so can't debug and fix this.

The game most likely launches some other processes from its install folder and doesn't close them down properly after the game itself exits. Xbox plugin would then detect game as still running because Xbox plugin detects game as running as long as some exe from game's install folder is running (this is the most reliable way of doing it for UWP/GamePass games).

**Comment by K900** [comment]
I don't think it does, but I will double check. The game is on Game Pass if you have that, or I'd be happy to poke around myself.

**Comment by JosefNemec** [comment]
Should be fixed by tracking changes made in never version of Playnite.
